sycl/test-e2e/Matrix/runtime_query_spr.cpp

```diff
@@ -0,0 +1,54 @@
+// REQUIRES: cpu
```

Comment:
do we need REQUIRES:cpu here.
Since you are using 
"(!dev.ext_oneapi_architecture_is(sycl::ext::oneapi::experimental::architecture::intel_cpu_spr))"
I was thinking we can merge the twp spr and pvc tests together and remove `REQUIRES`. Can the test make such distinction?

Reply:
The main benefit of `REQUIRES` is that the test will be reported as `UNSUPPORTED`/`skipped` and QA could detect if it's not running at all.

Having the check inside the test results in `PASS` on platforms that don't matter and might give us false confidence.